### DISCUSSION TITLE
Photos are stored to Pictures directory rather than project DCIM directory

### DISCUSSION THREAD

**Discussion by northwestview** [opengisch/QField#2731] [Q&A]
It appears that in QField 2 photos that were once stored in a DCIM subdirectory of the project directory are now stored in \Internal storage\Android\data\ch.opengis.qfield\files\Pictures directory with a different name, prefixed with QFieldPicture plus a datetime stamp. A file with the expected name is created in the DCIM direcotry, but it cannot be opened or copied from the Android filesystem to a Windows PC. Since the files have a different name than the actual images stored under Pictures, it is almost impossible to reconcile once copied back to the QGIS project. I had all of the photo attachment working correctly in the attribute form in v1.9 and earlier. The photos do appear when the feature is selected within QField but I cannot reference them in QGIS as the photo name under Pictures is different than the photo name stored in the feature. 
For example, the picture file name and path is stored as 
Android\data\ch.opengis.qfield\files\QField\<project directory>/DCIM/JPEG_20220405220229213.jpg 
in the feature, and an uncopyable, unopenable file of that name is stored at that location, but the actual image is stored under 
Android\data\ch.opengis.qfield\files\Pictures\QFieldPicture20220405_1502292415575831595130566jpg
and there is no extension.
Is there a new way to manage photo collection that has not been documented yet?

**Comment by suricactus** [comment]
Can you share a test project, I failed to reproduce your issue on the Simple Bee test project.

**Comment by northwestview** [comment]
After reinstalling the app and the project it stored the images correctly
in the project directory.

**Reply by suricactus** [reply]
Glad it works now!

**Reply by deepnpisgah** [reply]
I have somewhat the same issue. My last trip to the field I added several features with photos. For some reason the photos didn't push to the cloud along with the rest of the changes. In QGIS the features show only a path to the picture, no picture. In QField I can see the photos with the feature but they are stored the same as described originally in this post. Going back to April all the pictures are stored this same way, but I didn't notice as they were syncing with the cloud and importing into the QGIS project as expected. But now I've got a dozen or so images that are not synced and no apparent way to copy them from the android device into my QGIS project.  I am running QField 2.1.2 - Bumblebee (1c22b0 v2.1.2)
